ledger/acctupdates_test.go

```diff
@@ -526,7 +528,11 @@ func TestAcctUpdates(t *testing.T) {
 		accts = append(accts, newAccts)
 		rewardsLevels = append(rewardsLevels, rewardLevel)
 
-		checkAcctUpdates(t, au, 0, i, accts, rewardsLevels, proto)
+		// checkAcctUpdates is kind of slow because of amount of data it needs to compare
+		// instead, compare at start, end in between approx 10 rounds
+		if i == start || i == end-1 || crypto.RandUint64()%10 == 0 {
```

Comment:
could you avoid using a real random number here and make it a pseudo random ? just so it will be consistent ?

Reply:
I intentionally made it truly random to cover different intervals across multiple runs